### PR TITLE
Create nauplius.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/79607/style-request-nauplius

### CODE REVIEW

File: nauplius.csl [hunk 1]
```diff
@@ -0,0 +1,489 @@
...
+                <text variable="collection-title"/>
+              </else>
+            </choose>
+          </group>
+        </group>
+      </if>
+      <else-if type="chapter" match="any">
+        <group delimiter=", ">
+          <choose>
+            <if variable="ISBN ISBN" match="any">
```
Comment (adam3smith) [inline nauplius.csl]: I don't understand the test here -- don't see why a book chapter with an ISBN would be cited differently?

File: nauplius.csl [hunk 2]
```diff
@@ -0,0 +1,489 @@
...
+            <date-part name="year" range-delimiter="/"/>
+          </date>
+        </group>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="editor">
+    <names variable="editor translator">
+      <name and="text" initialize-with="."/>
+      <label form="short" strip-periods="true" prefix=" (" suffix=")"/>
```
Comment (adam3smith) [inline nauplius.csl]: You'll probably want the strip-periods on all names labels then, no?

**Comment by adam3smith** [comment]
Two quick comments -- this again looks a bit like a style where the template chose perhaps wasn't ideal, but might still be faster to just fix some of the weirdness in the style. Thanks!

**Comment by POBrien333** [comment]
Yup, way too complex a style for what it does, and cumbersome at that.
Thanks for spotting the issues.

**Comment by adam3smith** [comment]
Great, thanks!

**Pull request merged**